traefik/Chart.yaml

```
@@ -2,10 +2,10 @@ apiVersion: v2
 name: traefik
 description: A Traefik based Kubernetes ingress controller
 type: application
-version: 27.0.0
+version: 28.0.0-rc1
```

```suggestion
version: 28.0.0-rc.1
```
Per Helm guidelines/conventions (https://helm.sh/docs/topics/charts/#charts-and-versioning)

---

\o I missed that one. Many thanks  :+1: !
Fixed with 00eca8f